crates/anstyle-roff/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "anstyle-roff"
+version = "0.1.0"
+description = "Adapt between anstyle and roff"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/rust-cli/anstyle.git"
+homepage = "https://github.com/rust-cli/anstyle"
+categories = ["command-line-interface"]
+keywords = ["ansi", "terminal", "roff"]
+edition = "2021"
```

Comment:
When #32 lands, can you update this to use workspace inheritance?

Reply:
yes I'll do that.

Reply:
Merged